lib/Zonemaster/CLI.pm

```diff
@@ -358,9 +356,9 @@ sub run {
     }
 
     # These two must come after any profile from command line has been loaded
-    # to override the profile setting
-    Zonemaster::Engine::Profile->effective->set( q{net.ipv4}, 0+$self->ipv4 );
-    Zonemaster::Engine::Profile->effective->set( q{net.ipv6}, 0+$self->ipv6 );
+    # to override the profile setting, but only if selected.
+    Zonemaster::Engine::Profile->effective->set( q{net.ipv4}, 0+$self->ipv4 ) if defined ($self->ipv4);
+    Zonemaster::Engine::Profile->effective->set( q{net.ipv6}, 0+$self->ipv6 ) if defined ($self->ipv6);
```

Comment:
Could you update this to use `if (...) { ... }` syntax? This is in line with the rest of the code. Also, the structure of the code is more apparent to the reader that way.

I do realize there are a couple of instances of trailing if statements in this file, but adding more of them is a mistake from a readability point of view.

Reply:
Is that only your personal opinion that trailing "if" is always less readable, or do you know of any studies of that supports that? In that case we get three lines, instead of one, and more separators. And the condition is short.

Reply:
Are you being serious?

https://www.squirrel.nl/pub/PBP_refguide-1.02.00.pdf